api/src/main/java/com/nicknathanjustin/streamercontracts/users/UserServiceImpl.java

```diff
@@ -25,10 +23,7 @@ public UserModel createUser(@NonNull final String twitchUsername) {
 
     @Override
     public UserModel getUser(@NonNull final String twitchUsername) {
-        List<UserModel> users = userModelRepository.findByTwitchUsername(twitchUsername);
```

Comment:
Is there a reason for removing this? I know Justin said it is good practice to not rely on the "unique" database constraint and also check the result you are getting back from the database. I don't see any harm in keeping this code.

Reply:
So, we do need this code. We need the check if there is no user in the database. We also should keep the assert. We don't need to explicitly write a unit test to test if the assert fails, because our other unit tests will fail if the assert doesn't hold.